openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -130,6 +133,22 @@ def create_account_and_fetch_profile(self, host='microsite.example.com'):
         user = User.objects.get(username=self.username)
         return user.profile
 
+    @override_waffle_flag(PASSWORD_UNICODE_NORMALIZE, active=True)
+    def test_create_account_and_normalize_password(self, host='microsite.example.com'):
+        """
+        Test that unicode normalization on passwords is happening when a user registers.
```

Comment:
Similar to an above test, I would find it helpful if you added in a comment about how you're testing this by using an "incorrect" normalization to start and then ensuring it is normalized to what it should be later.